_posts/2018-06-25-CSharp-wrapper-released.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: C# wrapper released
+tags: news
+---
+
+Eric Freed has released a complete, cross-platform, managed wrapper called 
+[glfw-net](https://github.com/ForeverZer0/glfw-net) which covers GLFW 3.2 
+library for creating native windows with an OpenGL context.
```

Comment:
This last line breaks the pattern from previous similar posts. Should we do it this way from now on?

Reply:
My bad, didn't notice the pattern. I have no issues if you would like to keep consistent with previous posts.

Reply:
@elmindreda Is this pull request okay as it is? or do you prefer it to follow the same pattern from previous posts?

Reply:
I'd prefer to stick to the previous pattern.